app/build.gradle

```diff
@@ -29,12 +33,33 @@ android {
         versionCode rootProject.ext.androidVersionCode
         versionName rootProject.ext.androidVersionName
     }
+
+    signingConfigs {
+        signedRelease {
+            storeFile file(project["signingKeystore"])
+            storePassword ""
+            keyAlias ""
+            keyPassword ""
+        }
+    }
+
+    if (project.hasProperty("signingKeystore") && project.hasProperty("signingKeystorePassword") &&
+            project.hasProperty("signingKeyAlias") && project.hasProperty("signingKeyPassword")) {
+        android.signingConfigs.signedRelease.storePassword = project.getProperties().get("signingKeystorePassword")
```

Comment:
I think that using project.property("....") is shorter here.

Reply:
What I meant was that property is used after the if. Instead of getProperties().
The hasProperty for the if condition is still necessary as the property call expects the property to exist. See also the failed build.
Sorry for the inconvenience.